source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,64 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', () => {
+      const leaveAppRunning = Boolean(global.config.get('system.leaveAppRunning'))
+      if (leaveAppRunning) {
+        if (process.platform === 'darwin') {
+          if (this._tray == null) {
+            let basepath = app.getAppPath()
+            this._tray = new Tray(path.join(__dirname, './assets/icons/128x128.png'))
```

Comment:
On MacOS tray icon should be 22x22 with a white logo and transparent background as per the [tray design](https://github.com/UofA-SPI21-team7/zettlr-assignment/wiki/Tray-Design).

I suggest put the following code. Later #17 can install and setup the icon.
``` typescript
            this._tray = new Tray(path.join(__dirname, './assets/icons/22x22_white.png'))
```

Reply:
No need to create a new 22x22_white.png. There is a 22x22_white.png in #42